morphia/src/main/java/org/mongodb/morphia/IndexOptionsBuilder.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2016 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mongodb.morphia;
+
+import org.mongodb.morphia.annotations.Collation;
+import org.mongodb.morphia.annotations.Index;
+import org.mongodb.morphia.annotations.IndexOptions;
+import org.mongodb.morphia.annotations.Indexed;
+
+class IndexOptionsBuilder extends AnnotationBuilder<IndexOptions> implements IndexOptions {
```

Comment:
@Index applies to classes and @Indexed applies directly to fields.  Other than that, i'm not sure i understand what you're asking.